src/core/dbus-execute.c

```diff
@@ -563,7 +563,7 @@ static int property_get_bind_paths(
         assert(property);
         assert(reply);
 
-        ro = !!strstr(property, "ReadOnly");
+        ro = strstr(property, "ReadOnly");
```

Comment:
you are assigning a pointer to a bool here. Does this even compile?

I mean I am fine with dropping !! when we are converting something that is already a bool-like type to a bool, but this line above is really ugly...